register.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>register form</title>
+</head>
+<body>
+	<form>
+		<table>
+			<tr>
+				<td>

```

Comment:
I guess code is not formatted properly. try to make code formatted before commit